unassigner/command.py

```diff
@@ -27,16 +27,23 @@ def main(argv=None):
         "--output_dir",
         help=(
             "Output directory (default: basename of query sequences FASTA "
-            "file, plus '_unassigned')"
+            "file, plus '_unassigned'. Note that it will be in the same "
+            "directory as the query sequences FASTA file)."
         ),
     )
     p.add_argument(
         "--type_strain_fasta",
```

Comment:
The logic for handling both `--type_strain_fasta` and `--db_dir` is complicated, and I'm not sure it's possible to make it behave simply. Recommend removing or deprecating `--type_strain_fasta` in favor of `--db_dir`.

It might be possible to remove `--type_strain_fasta` and replicate the old behavior if you have the user set `--db_dir=.`.